community/sway/etc/skel/.config/foot/foot.ini

```diff
@@ -0,0 +1,3 @@
+include=~/.config/foot/foot-theme.ini
```

Comment:
Having this in the skel will override it when running `skel` which defeats your purpose, doesn't it? shouldn't we just keep the example ini file and extend the config file hierarchy in [foot.sh](https://github.com/manjaro-sway/desktop-settings/blob/sway/community/sway/usr/share/sway/scripts/foot.sh)?

1. .config/foot/foot.ini
2. .config/foot/foot-theme.ini
3. /usr/share/sway/templates/foot.ini